app/scripts/modules/core/projects/dashboard/cluster/projectCluster.directive.spec.js

```diff
@@ -121,20 +121,14 @@ describe('Directives: projectCluster', function () {
     var elem = $compile(html)($scope);
     $scope.$digest();
 
-    let upSelector = ' span[ng-if="container.up"]:eq(0)';
-
     expect(elem.find('.cluster-name').text().trim()).toBe('foo-*');
     expect(elem.find('.cluster-health:eq(0)').text().trim()).toBe('2 Applications');
-    expect(elem.find('.rollup-summary health-counts' + upSelector).text().trim()).toBe('4');
```

Comment:
Haha, yes. I removed this test because it's not testing the projectCluster directive, it's testing that the HealthCounts component does what it's supposed to.